.changeset/six-points-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
+---
+
+fix(SelectInput): single select value clear
```

Comment:
Let's also add a line on what works now for clearing the input value (eg. pass empty string)

Reply:
done